public/views/viewBuildError.html

```diff
@@ -61,141 +62,36 @@
-
-        <tr ng-if="error.argumentfirst">
-          <th class="measurement" style="width: 1%">Command</th>
-          <td>
-            <div style="margin-left: 25px; text-indent: -25px;">
-              <span id="showarguments_{{error.id}}" ng-hide="error.showArguments">
-                <span style="cursor: pointer" ng-click="error.showArguments = ! error.showArguments">
-                  [+]
-                </span>
-                <span class="nobr">"<font class="argument">{{error.argumentfirst}}</font>"</span>
-              </span>
-
-              <span id="argumentlist_{{error.id}}" ng-show="error.showArguments">
-                <span style="cursor: pointer" ng-click="error.showArguments = ! error.showArguments">
-                  [-]
-                </span>
-                <span class="nobr">"<font class="argument">{{error.argumentfirst}}</font>"</span>
-                <span ng-repeat="argument in error.arguments track by $index">"<font class="argument nobr">{{argument}}</font>" </span>
-              </span>
-            </div>
-          </td>
-        </tr>
-
-        <tr ng-if="error.workingdirectory">
-          <th class="measurement" style="width: 1%">
-            Directory
-          </th>
-          <td>
-            {{error.workingdirectory}}
-          </td>
-        </tr>
-
-        <tr ng-if="error.exitcondition">
-          <th class="measurement">
-            <span class="nobr"> Exit Condition </span>
-          </th>
-          <td>
-            {{error.exitcondition}}
-          </td>
-        </tr>
-
-        <tr ng-if="error.stdoutput">
-          <th class="measurement">
-            <span class="nobr"> Standard Output </span>
-          </th>
-          <td>
-            <pre class="compiler-output" name="stdout">{{error.stdoutput}}</pre>
-          </td>
-        </tr>
-
-        <tr ng-if="error.stderror">
-          <th class="measurement">
-            <span class="nobr"> Standard Error </span>
-          </th>
-          <td>
-            <pre class="compiler-output" name="stderr">{{error.stderror}}</pre>
-          </td>
-        </tr>
+      <!-- If it's a parent build, show build errors in an abbreviated view grouped by subproject -->
+      <table ng-if="cdash.parentBuild" border="0" cellpadding="4" cellspacing="0" width="100%" class="tabb">
+        <thead>
+          <tr class="table-heading">
+            <th align="left" width="15%">
+              Subproject
+            </th>
+            <th align="left">
+              Errors
+            </th>
+          </tr>
+        </thead>
+
+        <tbody>
+          <tr ng-repeat="(subprojectName, errors) in cdash.errors">
+            <td style="vertical-align:top">{{subprojectName}}</td>
+            <td><a href="#" ng-click="showErrors = !showErrors">{{errors.length}} {{cdash.errortypename}}s found.</a>
+              <div ng-hide="!showErrors">
+                <ng-include src="'views/partials/buildError.html'" ng-repeat="error in errors"/>
```

Comment:
It's been my experience that using ``ng-include`` inside ``ng-repeat`` causes the page to render slowly.  I recommend using a directive here instead.  [Here's an example to get you started](https://github.com/Kitware/CDash/blob/master/public/js/controllers/index.js#L518).